zeebe/engine/src/test/java/io/camunda/zeebe/engine/util/EngineRule.java

```diff
@@ -184,7 +184,7 @@ protected void after() {
   }
 
   public void start() {
-    start(StreamProcessorMode.PROCESSING, true);
+    start(environmentRule.getStreamProcessorMode(), true);
```

Comment:
🤯 nice catch @danielkelemen (cc @remcowesterhoud). How long is this maybe already broken?

Reply:
The [`ContinuouslyReplayTest`](https://github.com/camunda/camunda/blob/main/zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/ContinuouslyReplayTest.java#L27-L30) used only the `.withStreamProcessorMode(StreamProcessorMode.REPLAY);` builder method and that gets overridden by the default start parameter in the `EngineRule`.